src/content/operations/nat-traversal/_index.en.md

```diff
@@ -0,0 +1,386 @@
+
+```bash
+kubectl annotate node $GWA --kubeconfig A gateway.submariner.io/natt-discovery-port=4491
+kubectl annotate node $GWA --kubeconfig A gateway.submariner.io/udp-port=4501
+kubectl annotate node $GWB --kubeconfig B gateway.submariner.io/natt-discovery-port=4492
+kubectl annotate node $GWB --kubeconfig B gateway.submariner.io/udp-port=4502
+kubectl annotate node $GWC --kubeconfig C gateway.submariner.io/natt-discovery-port=4491
+kubectl annotate node $GWC --kubeconfig C gateway.submariner.io/udp-port=4501
+kubectl annotate node $GWD --kubeconfig D gateway.submariner.io/natt-discovery-port=4492
+kubectl annotate node $GWD --kubeconfig D gateway.submariner.io/udp-port=4502
+
+# restart the gateways to pick up the new setting
+for cluster in A B C D;
+do
+  kubectl delete pod -n submariner-operator -l app=submariner-gateway --kubeconfig $cluster
+done
+```
+
+{{% notice warning %}}
+If **HA** is configured on the on-premises clusters, **each gateway** behind the routers
+**should have a dedicated UDP port**. For example if we had 2 clusters and 2 gateways on each network,
+4 ports would be necessary.
+{{% /notice %}}
+
+#### Router Port Mapping for Multiple On-Oremises
+
+Under this configuration it's important to map the UDP ports on the 2.1.1.1 router to the private
+IPs of the gateways.
+
+##### On the 2.1.1.1 router
+
+| External IP | Port | Internal IP   | Port | Protocol |
+|:------------|:-----|:--------------|:-----|:---------|
+| 2.1.1.1     | 4501 | 192.168.0.4   | 4501 | UDP      |
+| 2.1.1.1     | 4491 | 192.168.0.4   | 4491 | UDP      |
+| 2.1.1.1     | 4502 | 192.168.128.4 | 4502 | UDP      |
+| 2.1.1.1     | 4492 | 192.168.128.4 | 4492 | UDP      |
+
+##### On the 1.1.1.1 router
+
+| External IP | Port | Internal IP   | Port | Protocol |
+|:------------|:-----|:--------------|:-----|:---------|
+| 1.1.1.1     | 4501 | 10.0.0.1      | 4501 | UDP      |
+| 1.1.1.1     | 4491 | 10.0.0.1      | 4491 | UDP      |
+| 1.1.1.1     | 4502 | 10.1.0.1      | 4502 | UDP      |
+| 1.1.1.1     | 4492 | 10.1.0.1      | 4492 | UDP      |
+
+{{% notice note %}}
+Without port mapping it's entirely possible that the connectivity will be established without
+issues, this happens due to the fact that route's NAT will not generally modify the src port
+of the outgoing UDP packets, and future packets arriving this port will be redirected to the
+internal IP which initiated connectivity, but if the 2.1.1.1 router randomizes the source port on
+NAT, or if other applications on the internal network were already using the 4501-4502,4491-4492
+ports, the remote ends would not be able to contact gateway C or D over the expected ports.
+{{% /notice %}}
+
+### Double NAT Traversal
+
+<!-- source: https://app.diagrams.net/#G1VTxw02pmAgE1WeGX5vWfSswZNciK-5AC -->
+![on-premises vs on-premises](/images/natt/double_nat_scenario.svg)
+
+Let's see why in the following sections.
```

Comment:
@mangelajo is this how you start this section or maybe you omitted some lines before?

Reply:
Hmm I probably should elaborate as for the other scenarios. thanks